README.md

```diff
@@ -1,112 +1,36 @@
 # VSCode Trace Server extension
 
-* This is a TypeScript extension, officially named `vscode-trace-server`.
-* It is meant as companion to the [vscode-trace-extension][vscode-trace-extension].
-* It registers `Trace Server:` start/stop commands, for a default instance locally.
-* It depends on the [tsp-typescript-client][client] for server health check purposes.
+This is a companion extension to the Trace Viewer for VSCode. It makes it easier to control a default local TSP ([Trace Server Protocol][tsp]) enabled trace server, by providing `Trace Server:` start/stop commands.
```

Comment:
> we should reference and to how to get Trace Compass trace server (similar how you did it in the vscode-trace-extension release README:

To try to avoid duplication, I have instead used a link to the relevant section of the Trace Viewer's README. Let me know if it's ok with you.

> Besides should we mention that it registers as start handler of the vscode-trace-extension API? Or is that implementation specific?

I think for end-users this would be implementation details. Such info would probably be more suited to be in `README-dev.md`.  I propose we improve that documentation separately, if it's something we want - I just did a minimal pass as part of this PR.

Reply:
I'm ok with the external reference. We just need to remember to update this reference if the location changes in the other repo.